docs/block-api/attributes.md

```diff
@@ -55,26 +55,17 @@ Use `html` to extract the inner HTML from markup.
 // { "content": "The inner text of the <strong>figcaption</strong> element" }
 ```
 
-### `children`
-
-Use `children` to extract child nodes of the matched element, returned as an array of virtual elements. This is most commonly used in combination with the `RichText` component.
-
-_Example_: Extract child nodes from a paragraph of rich text.
+Use the `multiline` property to extract the inner HTML for the use in `RichText` with the `multiline` prop.
 
 ```js
 {
 	content: {
-		source: 'children',
-		selector: 'p'
+		source: 'html',
+		multiline: 'p',
+		selector: 'blockquote',
 	}
 }
-// {
-//   "content": [
-//     "Vestibulum eu ",
-//     { "type": "strong", "children": "tortor" },
-//     " vel urna."
-//   ]
-// }
+// { "content": "<p>Quoted text</p><p>And more</p>" }
```

Comment:
Maybe using an example like `"<p>Quoted text</p><p>Second line</p>"` might make this a bit more immediately obvious.